test/langtools/tools/javac/versions/Versions.java

```diff
@@ -93,6 +94,7 @@ static enum SourceTarget {
         TWENTY(false,    "64.0", "20"),
         TWENTY_ONE(false,"65.0", "21"),
         TWENTY_TWO(false,"66.0", "22"),
+        TWENTY_THREE(false,"67.0", "23"),
```

Comment:
As alignment is broken anyway, let's add space
```suggestion
        TWENTY_THREE(false, "67.0", "23"),
```

Reply:
If you're going to add the space, I'd consider establishing new alignment at TWENTY to accommodate the eventual space needed for "THREE", "SEVEN", and "EIGHT".  You wouldn't need to reset alignment for quite some time after that.